public/styles/common.css

```diff
@@ -351,17 +351,17 @@ ul.unmarkedList > li:hover,
 
 .light { opacity: 0.8; }
 
-#registerPackage {
+#inputForm {
```

Comment:
There are no cases with more than 1 input form on the page, but yes, making it `.inputForm` looks better. I'll fix it.